### DISCUSSION TITLE
DB Behaviour on OCPP tags deletion

### DISCUSSION THREAD

**Discussion by fcojacuna** [steve-community/steve#1449] [Q&A] [answered]
It seems that if you erase an OCPP tag all the transactions related to that tag are also deleted in the DB, is this on purpose?

**Comment by goekay** [comment, marked as answer]
yes. there is even a hint about this: 
<img width="491" alt="Screenshot 2024-05-20 at 13 51 56" src="https://github.com/steve-community/steve/assets/3861031/0c4dab94-a776-4d94-8d12-858b6bf97166">

**Reply by fcojacuna** [reply]
Clear like water, can't believe I didnt see that. Thank you so much. Is there a wayy to change this behaviour?
